src/env-config.js

```diff
@@ -22,6 +22,8 @@ const envConfig = {
       && ['false', 'disable'].includes(process.env.VUE_APP_WEBTRIT_NAVIGATION_DRAWER_MINIMIZING.toLowerCase())
   ),
 
+  webtritRecordingPlayButtonDisable: process.env.VUE_APP_WEBTRIT_RECORDING_PLAY_BUTTON_DISABLE.toLowerCase() === 'true',
```

Comment:
The variable `process.env.VUE_APP_WEBTRIT_RECORDING_PLAY_BUTTON_DISABLE` returns `undefined` if the environment variable isn't set. Attempting to call `toLowerCase()` on `undefined` will result in a `TypeError`.
This is why we need to handle it similarly to `webtritNavigationDrawerMinimizing`.